### ISSUE TITLE
The revenue per city, per borough, and per taxi.

### ISSUE THREAD

**Issue by ElijahsParadigm** [ElijahsParadigm/Blue_Rabbit#9]
Analyze yellow and green taxi data from January 2023 till March 2023, alongside the weather for each of the trips at the pickup and the arrival.
1. The revenue per city, per borough, and per taxi.
2. Your insights from the analyzed data.

**Comment by ElijahsParadigm** [comment]
Our cloud storage solution will be Microsoft Azure Blob. I will work on moving the raw data to the cloud.

**Comment by ElijahsParadigm** [comment]
Created a storage container in Microsoft Azure Blob, and pushed all raw data to the container.

**Comment by ElijahsParadigm** [comment]
Completed the ETL process and pushed transformed data back to the storage container. Linked storage container to the data warehouse(Snowflake). I accessed the clean data using Tableau. I created a heat map with the top 10 revenue per city and a bar chart with the highest revenue per borough. The overall analysis: Highest Green Taxi Revenue per city - Upper East Side North; Highest Yellow Taxi Revenue Per City - JFK Airport, Queens; Highest Revenue Per Borough - Manhattan.
